File: app/src/main/java/in/woloo/www/dashboard/ui/home/HomeViewPresenterInterface.java
package in.woloo.www.dashboard.ui.home;

import in.woloo.www.dashboard.ui.home.model.NearByStoreResponse;
import in.woloo.www.networksUtils.NetworkAPICallModel;
import in.woloo.www.splash.AuthConfigResponse;

public interface HomeViewPresenterInterface {
  void onGetNearByStore(NearByStoreResponse nearByStoreResponse, NetworkAPICallModel networkAPICallModel);
  void authConfigSuccess(AuthConfigResponse authConfigResponse);
}

File: app/src/main/java/in/woloo/www/application_kotlin/interfaces/HomeViewPresenterInterface.kt
package `in`.woloo.www.application_kotlin.interfaces

import `in`.woloo.www.application_kotlin.model.server_response.AuthConfigResponse
import `in`.woloo.www.application_kotlin.model.server_response.NearByStoreResponse
import `in`.woloo.www.application_kotlin.api_classes.NetworkAPICallModel

interface HomeViewPresenterInterface {
    fun onGetNearByStore(
        nearByStoreResponse: NearByStoreResponse?,
        networkAPICallModel: NetworkAPICallModel?
    )

    fun authConfigSuccess(authConfigResponse: AuthConfigResponse?)
}
